src/conf_mode/protocols_isis.py

```diff
@@ -203,6 +203,29 @@ def verify(isis):
         if list(set(global_range) & set(local_range)):
             raise ConfigError(f'Segment-Routing Global Block ({g_low_label_value}/{g_high_label_value}) '\
                               f'conflicts with Local Block ({l_low_label_value}/{l_high_label_value})!')
+        
+    # Check for a blank or invalid value per prefix
+    if dict_search('segment_routing.prefix', isis):
+        for prefix, prefix_config in isis['segment_routing']['prefix'].items():
+            if 'absolute' in prefix_config:
+                if prefix_config['absolute'].get('value') is None:
+                    raise ConfigError(f'Segment routing prefix {prefix} absolute value cannot be blank.')
+            elif 'index' in prefix_config:
+                if prefix_config['index'].get('value') is None:
+                    raise ConfigError(f'Segment routing prefix {prefix} index value cannot be blank.')
+
+    # Check for explicit-null and no-php-flag configured at the same time per prefix
+    if dict_search('segment_routing.prefix', isis):
+        for prefix, prefix_config in isis['segment_routing']['prefix'].items():
+            for segment_type in ['absolute', 'index']:
+                try:
```

Comment:
We have almost zero `try:/except` blocks in `validate()`,  please simplye probe if the key is present.

```python
if segment_type not in prefix_config:
    continue
```

Also `explicit_null = isis['segment_routing']['prefix'][prefix][segment_type]` can be shortened to `explicit_null = prefix_config[segment_type]`

Reply:
I think I've reduced the complexity, and the matching down. I removed the try block as well.

See if you like it my good sir :)